app/controllers/admin/places_controller.rb

```diff
@@ -11,8 +11,13 @@ def new
   end
 
   def index
-    @search = search_places_by_parameters(params[:q])
+    query_params = (request[:format] == 'json') ? JSON.parse(params['q']) : params[:q]
```

Comment:
__[rubocop]__
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)